.github/ISSUE_TEMPLATE/refactor_tickets.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 🔧 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🔧 [Refactor]
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-02-04T14:23:29Z" title="Tuesday, February 4th 2020, 3:23:29 pm +01:00">Feb 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -0,0 +1,18 @@
+---
+name: 🐛 Refactor ticket
+about: Help us improve our code by refactoring it.
+labels: refactor
+title: 🐛 [Refactor] 
```


```suggestion
title: 🔧 [Refactor] 
```

same here – feel free to choose another one, of course! it was just the first one that came to mind...